underscore.js

```diff
@@ -94,6 +93,30 @@
     return cb(value, context, Infinity);
   };
 
+  // Similar to ES6's rest param (http://ariya.ofilabs.com/2013/03/es6-and-rest-parameter.html)
+  // This accumulates the arguments passed into an array, after a given index.
+  var restArgs = function(func, startIndex) {
+    startIndex = startIndex == null ? func.length - 1 : +startIndex;
```

Comment:
I think the usefulness of not having to give the `startIndex` every times outweighs this trip-up. A lot of what I imagine using it for is in parse-time. You would then bind the returned function.

```javascript
var args = _.restArgs(function(args) { return args; }).bind(null);
```